client/blocks/comment-detail/index.jsx

```diff
@@ -129,6 +131,18 @@ export class CommentDetail extends Component {
 
 	exitReplyState = () => this.setState( { isReplyMode: false } );
 
+	getPostUrl = () => {
+		const { commentStatus, postId, postUrl, siteSlug } = this.props;
+
+		if ( ! config.isEnabled( 'comments/management/post-view' ) ) {
+			return postUrl;
+		}
+
+		const status = 'unapproved' === commentStatus ? 'pending' : commentStatus;
```

Comment:
I think I've seen this in a couple places, and it's one of those magic string that can get lost really fast. Maybe we should do that transformation on the state reducer or the selector.

Reply:
Want to add a task so we don't forget to do this refactor?